netlify.toml

```diff
@@ -1,2 +1,2 @@
 [build]
-  ignore = "git diff --quiet HEAD^ HEAD packages/docs-website/"
\ No newline at end of file
+  ignore = "git diff --quiet HEAD^ HEAD packages/docs-website/ packages/nitro-protocol"
```

Comment:
How does this command work? What's its purpose?

Reply:
It wires up netlify's ["ignore a build"](https://docs.netlify.com/configure-builds/file-based-configuration/#ignore-builds) configuration to the exit code of a git command. It's purpose is to skip the build *unless* the docs-website has changed OR nitro-protocol has changed. It should prevent a lot of unnecessary deploy previews being built.

Reply:
> It wires up netlify's ["ignore a build"](https://docs.netlify.com/configure-builds/file-based-configuration/#ignore-builds) configuration to the exit code of a git command. It's purpose is to skip the build _unless_ the docs-website has changed OR nitro-protocol has changed. It should prevent a lot of unnecessary deploy previews being built.

Does `.toml` support comments? This could be useful information to live above the `ignore = ` line.